modules/ROOT/pages/cipher-suites.adoc

```diff
@@ -33,6 +33,20 @@ If you select a *TLS Version* value of *TLS 1.3*, keep the default selection, wh
 
 If you add a cipher that's not supported by the Java runtime, you receive a warning, but this does not prevent you from saving the TLS context.
 
+== TLS Support on Anypoint Runtime Fabric
+
+The TLS 1.3 protocol is enabled by default for those clients that support it.
+
+OpenSSL 1.1.1 and TLS 1.3 provide the following security enhancements and performance improvements over TLS 1.2:
+
+* Double or more than double TLS 1.2 connection performance throughput boost over an earlier Runtime Fabric internal load balancer running on OpenSSL 1.0.2
```

Comment:
At first read it sounded confusing. Maybe what you are trying to say could be reordered as:

> Double or more than double connection performance boost of your network throughput compared to an earlier Runtime Fabric internal load balancer running OpenSSL 1.0.2 and TLS 1.2